### PR TITLE
Use startup_time from system.connected

### PR DESCRIPTION
**Pull request by maciejhirsz**
Bonus: removed `level` from messages, so they can be dropped in substrate.

### CODE REVIEW

File: backend/src/node/connector.rs [hunk 1]
```diff
@@ -161,7 +161,7 @@ impl StreamHandler<Result<ws::Message, ws::ProtocolError>> for NodeConnector {
 
         match serde_json::from_slice(&data) {
             Ok(msg) => {
-                // info!("New node message: {}", std::str::from_utf8(&data).unwrap_or_else(|_| "INVALID UTF8"));
+                // log::info!("New node message: {}", std::str::from_utf8(&data).unwrap_or_else(|_| "INVALID UTF8"));
```
Comment (niklasad1) [inline backend/src/node/connector.rs:164]: can we remove the comment?

**Review by niklasad1 — COMMENTED** [review]
LGTM but I don't know all the details :)

**Comment by dvdplm** [comment]
> LGTM but I don't know all the details :)

Yeah, description is a bit lacking here. The context is https://github.com/paritytech/substrate/pull/7463 and https://github.com/paritytech/substrate/pull/7463#discussion_r515011072 I think.

File: backend/src/node.rs [hunk 2]
```diff
@@ -33,16 +33,20 @@ pub struct Node {
     location: Option<Arc<NodeLocation>>,
     /// Flag marking if the node is stale (not syncing or producing blocks)
     stale: bool,
-    /// Connected at timestamp
-    connected_at: Timestamp,
+    /// Unix timestamp for when node started up (falls back to connection time)
```
Comment (dvdplm) [inline backend/src/node.rs:36]: Should we really be falling back to connection time here? They are different things. Would switching to an `Option<Timestamp>` be very messy?
If we're ever interested in the time the node connected – and I think we're not – I think it should be a separate field.

**Review by dvdplm — CHANGES_REQUESTED** [review]
I think it would be better to not fall back to `connected_at`

File: backend/src/node.rs (hunk 2 above)
Comment (maciejhirsz) [reply backend/src/node.rs:36]: It's a fallback for older version which didn't send the field, which is also what the current functionality is. I believe at this point most nodes should be sending this over, to turn it into an `Option<Timestamp>` requires bumping protocol version at connection handshake with FE, and some changes there to visually account for `null`s.

Comment (dvdplm) [reply backend/src/node.rs:36]: I see. Can we fall back to `UNIX_EPOCH` or some other value "zero-ish" value?

Comment (maciejhirsz) [reply backend/src/node.rs:36]: This shows up as Uptime in the FE, I don't think showing 50 years of uptime is better than showing the time the node connected to telemetry at as fallback :)

**Review by dvdplm — APPROVED** [review]
Lovely, thank you! :)

**Review by niklasad1 — APPROVED** [review]

**Pull request merged**